src/ducks/balance/History.styl

```diff
@@ -1,9 +1,11 @@
 .HistoryChart
-    padding-bottom .5rem
     overflow-y auto
     background-color dodgerBlue
     margin 0
 
+.History .HistoryChart
+    padding-bottom .5rem
```

Comment:
Why moving this property in another selector? Is the goal to add more specificity?

Reply:
Yes because HistoryChart can be use on Balance & Transaction page

Reply:
Then maybe this padding should not be managed by `History`, but by the component that uses `History`. By passing a custom `className` to it for example.